python/lsst/analysis/drp/gatherResourceUsage.py

```diff
@@ -213,6 +215,10 @@ def build_quantum_graph(cls, metadata_refs):
             References to metadata datasets.  Non-metadata datasets are
             silently ignored, in order to make it easier to pass in more
             general dataset query results.
+        output_run : `str`
+            Name of the output RUN collection.
+        graph_metadata : `dict` [ `str`, `Any` ]
+            Graph metadata.
```

Comment:
I think either we say that output_run will be added to graph_metadata here (so we can ensure it exists) or else we declare that the output_run must be in metadata and then it doesn't need to be a parameter here. I know this is effectively a private API but it seems wrong to me that we can end up with the API being called without output_run ending up in the metadata.